### PR TITLE
PopupBridge – AppCompatActivity Overload

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

- Support AppCompatActivity overload for PopupBridge::newInstance

 ### Checklist

 - [x] Added a changelog entry

- @billwerges
- @sshropshire

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # PopupBridge Android Release Notes
 
+## unreleased
+* Support AppCompatActivity overload for PopupBridge::newInstance
+* Deprecate AppCompatActivity overload for PopupBridge::newInstance
```
Comment (hollabaq86) [inline CHANGELOG.md]: I do not understand the difference between these two lines...

Comment (hollabaq86) [reply CHANGELOG.md]: Should the first entry be support for FragmentActivity instead?

Comment (sshropshire) [reply CHANGELOG.md]: Yeah this is a brain typo haha. Will fix 🙏

File: PopupBridge/src/test/java/com/braintreepayments/popupbridge/PopupBridgeTest.java [hunk 2]
```diff
@@ -43,7 +45,7 @@
 @RunWith(RobolectricTestRunner.class)
 public class PopupBridgeTest {
 
-    private AppCompatActivity mActivity;
+    private FragmentActivity mActivity;
```
Comment (scannillo) [inline PopupBridge/src/test/java/com/braintreepayments/popupbridge/PopupBridgeTest.java]: Do we want to keep a test around that uses an `AppCompatActivity` in addition to the newly supported `FragmentActivity`?

File: PopupBridge/src/test/java/com/braintreepayments/popupbridge/test/TestActivity.java [hunk 3]
```diff
@@ -6,8 +6,9 @@
 
 import androidx.annotation.Nullable;
 import androidx.appcompat.app.AppCompatActivity;
```
Comment (scannillo) [inline PopupBridge/src/test/java/com/braintreepayments/popupbridge/test/TestActivity.java]:
```suggestion
```
If we aren't using `AppCompatActivity` can we delete this import?

**Comment by scannillo** [comment]
New changes look good to me! Maybe just update the PR description to remove the `deprecation` note.

**Pull request merged**